pkgs/development/libraries/libraw1394/default.nix

```diff
@@ -11,6 +11,7 @@ stdenv.mkDerivation rec {
   meta = { 
     description = "Library providing direct access to the IEEE 1394 bus through the Linux 1394 subsystem's raw1394 user space interface";
     homepage = "https://ieee1394.wiki.kernel.org/index.php/Libraries#libraw1394";
-    license = ["GPL" "LGPL"];
+    license = stdenv.lib.licenses.lgpl2;
```

Comment:
Our docs don't specify the meaning of list of licenses. Some committers mean dual licensing (i.e. you can choose) and some mean that different parts of the sources/package have different licenses.